mchf-eclipse/drivers/audio/cw/cw_gen.c

```diff
@@ -226,7 +226,7 @@ static const float sm_table[CW_SMOOTH_TBL_SIZE] =
 
 void CwGen_SetSpeed()
 {
-    ps.dit_time         = 1650/ts.keyer_speed;      //100;
+    ps.dit_time         = 1920/ts.keyer_speed;      //100;
 }
```

Comment:
Can't measure right now. But I think your measurements make sense when I look at the code.
I must admit, I don't see this whole thing a major issue but if we change something in this area, we should do it right.